<?xml version="1.0"?>
<launch>

	<group ns="base">
		<node name="f9p" pkg="ublox" type="ublox_ros">
			<param name="serial_port" value="/dev/ttyACM0"/>
			<param name="chain_level" value="0"/>
			<param name="rover_quantity" value="1"/>
 			<param name="local_host" value="192.168.0.111"/>
			<param name="local_port" value="16140"/>
			<param name="rover_host" value="192.168.0.202"/>
			<param name="rover_port" value="16145"/>
			<param name="log_filename" value="../data/udp_base_binary"/>
		</node> 
		<node pkg="rosbag" type="record" name="record" output="screen" args="-O ../data/udp_base.bag --all"/>
	</group>
</launch>
